README.md

```diff
@@ -1,4 +1,12 @@
-# pharmacy-first
+# Pharmacy First
+
+The COVID-19 pandemic has caused a significant backlog in hospital care, which has in turn contributed to an unprecedented increase in demand for GP appointments. In response, NHS England announced the [“Delivery plan for recovering access to primary care”](https://www.england.nhs.uk/publication/delivery-plan-for-recovering-access-to-primary-care/). A key element is the [Pharmacy First service](https://www.england.nhs.uk/primary-care/pharmacy/pharmacy-services/pharmacy-first/), which incorporates urgent supply of repeat medication, consultation for minor illnesses, and a clinical pathways service to assess for and manage seven common conditions, including supplying prescription-only medicines where appropriate. The plan also includes an expansion of the contraception and hypertension case-finding services. These services all aim to enable patients to access care in quicker and more convenient ways where otherwise this may have been more difficult or delayed due to the impacts of the pandemic.
+
+We plan to investigate the information sent from community pharmacies to patients' GP records following a consultation under the above services to describe how the services are utilised. Our findings will help us understand how the new services are helping patients to access care, particularly in light of the increased strain on GP appointments caused by the pandemic.
+
+We also plan to assess the possibility of linking additional datasets to OpenSAFELY to assess the potential impacts of the scheme on antimicrobial resistance (where bacteria develop ways to resist the drugs designed to kill them) and over the counter medicine sales (medicines available to buy without prescription).
+
```

Comment:
```suggestion
```

Reply:
I think I would focus on the data development work here, rather than the research project?